src/template/partials/footer.html

```diff
@@ -1,3 +1,52 @@
 <footer>
+  <!-- Start Footer -->
+  <div class="footer">
+  
+    <div class="container">
+        <div class="download">
+            <h3>Grow your business fast <br>with Sark SaaS Landing.</h3>  
+                <button name="Download" value="Download">Download Now</button>
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/a88831b0-e867-427b-b286-0c25eac37bbf)
1-space between div (download) and top section not equal 160px but equal 80px
and Why do you use padding and margin together? It is better that you use padding in section footer

Reply:
done